app/src/main/java/ingsw/group1/findmyphone/log/LogItemComparatorFactory.java

```diff
@@ -0,0 +1,108 @@
+package ingsw.group1.findmyphone.log;
+
+import androidx.annotation.NonNull;
+
+import java.util.Comparator;
+
+import ingsw.group1.findmyphone.event.EventOrder;
+
+/**
+ * @author Riccardo De Zen
+ */
+public class LogItemComparatorFactory {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'LogItemComparatorFactory' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/manual/RicDeZen/ingsw-group1-findmyphone/pullRequest?prid=5010214)